config/routes.rb

```diff
@@ -10,6 +10,7 @@
   resources :challenges, only: %i[index show], controller: 'ui/challenges' do
     resources :members, only: %i[create destroy], controller: 'ui/members'
   end
+  resources :tasks, only: :show, controller: 'ui/tasks'
```

Comment:
what if we will have two tasks from different challenges, but with the same title?

1. Should task.title be unique? 
2. Or it should be unique in the scope of a challenge as it is now, and we need to move tasks under challenges here

```suggestion
  resources :challenges, only: %i[index show], controller: 'ui/challenges' do
    resources :tasks, only: :show, controller: 'ui/tasks'
    resources :members, only: %i[create destroy], controller: 'ui/members'
  end
 
```

Reply:
We use `task.slug` as id and it is unique